src/measure_theory/constructions/borel_space.lean

```diff
@@ -1136,6 +1136,55 @@ lemma ae_measurable_restrict_of_antitone_on [linear_order β] [order_closed_topo
   ae_measurable f (μ.restrict s) :=
 @ae_measurable_restrict_of_monotone_on αᵒᵈ β _ _ ‹_› _ _ _ _ _ ‹_› _ _ _ _ hs _ hf
 
+private lemma measurable_set_of_mem_nhds_within_Ioi_aux [densely_ordered α]
```

Comment:
```suggestion
lemma measurable_set_of_mem_nhds_within_Ioi_aux [densely_ordered α]
```

Reply:
This lemma is strictly weaker than the next one (one assumption more), so I don't see the point in keeping it public.

Reply:
[Here](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/.E2.9C.94.20private.20and.20parameter/near/255174835) are my arguments to make no lemmas in the library private. 
But if you think this lemma should be private, you can merge this PR without making the change.